### PR TITLE
Added contact senors to the main wheels

### PR DESCRIPTION
**Pull request by ipa-kut**
This will enable wheel drop sensors event publishers https://github.com/yujinrobot/kobuki_desktop/pull/64

Addresses https://github.com/yujinrobot/kobuki_desktop/issues/62

### CODE REVIEW

File: kobuki_description/urdf/kobuki_gazebo.urdf.xacro [hunk 1]
```diff
@@ -184,6 +198,8 @@
         <cliff_detection_threshold>0.04</cliff_detection_threshold>
         <bumper_name>bumpers</bumper_name>
         <imu_name>imu</imu_name>
+        <wheel_left_sensor_name>wheel_left</wheel_left_sensor_name>
```
Comment (corot) [inline kobuki_description/urdf/kobuki_gazebo.urdf.xacro]: sensor name is wheel_drop; what about calling this:
`<wheel_drop_left_sensor_name>wheel_drop_left</wheel_drop_left_sensor_name>`
`<wheel_drop_right_sensor_name>wheel_drop_right</wheel_drop_right_sensor_name>`

File: kobuki_description/urdf/kobuki_gazebo.urdf.xacro [hunk 2]
```diff
@@ -9,6 +9,13 @@
       <kd>100.0</kd>
       <minDepth>0.001</minDepth>
       <maxVel>1.0</maxVel>
+      <sensor type="contact" name="wheel_left">
```
Comment (corot) [inline kobuki_description/urdf/kobuki_gazebo.urdf.xacro]: wheel_drop_left,  see below

File: kobuki_description/urdf/kobuki_gazebo.urdf.xacro [hunk 3]
```diff
@@ -9,6 +9,13 @@
...
+        <always_on>1</always_on>
+        <update_rate>50.0</update_rate>
+        <visualize>true</visualize>
+        <contact>
+          <collision>wheel_left_link_collision</collision>
+        </contact>
     </gazebo>
```
Comment (corot) [inline kobuki_description/urdf/kobuki_gazebo.urdf.xacro:20]: `</sensor>` missing

File: kobuki_description/urdf/kobuki_gazebo.urdf.xacro [hunk 4]
```diff
@@ -18,6 +25,13 @@
...
       <minDepth>0.001</minDepth>
       <maxVel>1.0</maxVel>
+      <sensor type="contact" name="wheel_right">
+        <always_on>1</always_on>
+        <update_rate>50.0</update_rate>
+        <visualize>true</visualize>
+        <contact>
+          <collision>wheel_right_link_collision</collision>
+        </contact>
     </gazebo>
```
Comment (corot) [inline kobuki_description/urdf/kobuki_gazebo.urdf.xacro:37]: same

**Review by corot — CHANGES_REQUESTED** [review]
Needs some fixes

**Comment by corot** [comment]
Hey @ipa-kut, I have retaken your work here, I hope you don't mind

**Review by corot — APPROVED** [review]

**Review by stonier — APPROVED** [review]

**Pull request merged**